### PR TITLE
Ubuntu 22.04 and ROS 2 Humble Install

### PR DESCRIPTION
**Pull request by hello-chintan**
This PR adds setup scripts for Ubuntu 22.04 and ROS 2 Humble. The scripts expect that ROS 2 Humble would be standalone so calibration files are sourced from $HELLO_FLEET_ID directory.

To test, follow the instructions on installing a fresh Ubuntu 22.04 OS and then execute the stretch_new_robot_install.sh script in stretch_install/factory/22.04 as:
```
cd ~/stretch_install/
./factory/22.04/stretch_new_robot_install.sh
```

### CODE REVIEW

File: docs/install_ubuntu_22.04.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+![](./images/20.04/installer_software.png)
+
+### Erase & Reinstall vs Install Alongside
+
+On the next page titled 'Installation type', you may choose between 'Erase disk and reinstall Ubuntu' or 'Install Ubuntu 22.04 alongside Ubuntu XX.04'. If you've already backed up data from the previous partition, or the previous partition is corrupted, select the erase & reinstall option. If you'd like to preserve your previous Ubuntu partition, select the alongside option. If you choose the alongside option, another screen will allow you to change the size of each partition. It's recommended to give each partition at least 50GB.
+
+Here's what the Erase & Reinstall option will look like, and an screenshot of the Install Alongside option is shown below.
+
+![](./images/18.04/erase_reinstall_disk.png)
```
Comment (hello-atharva) [inline docs/install_ubuntu_22.04.md:59]: Can we change this to 22.04/20.04?

File: docs/robot_install.md [hunk 2]
```diff
@@ -43,10 +47,11 @@ This step can be skipped if your robot had an existing software install on it. O
 
 ## Install Ubuntu
 
-Choose between the following guides based on whether you're installing Ubuntu 18.04 or Ubuntu 20.04 (see above for info on what software ships with each OS). Within these guides, you'll have the choice of whether to replace the previous OS partition or to install alongside it. If you choose to install alongside it, you'll also be able to choose the size of each partition on the hard drive.
+Choose between the following guides based on which version of Ubuntu you're installing ([see above](#why) for info on what software ships with each OS). Within these guides, you'll have the choice of whether to replace the previous OS partition or to install alongside it. If you choose to install alongside it, you'll also be able to choose the size of each partition on the hard drive.
 
- - [Ubuntu 18.04 Installation guide](./install_ubuntu_18.04.md).
- - [Ubuntu 20.04 Installation guide](./install_ubuntu_20.04.md).
+ - [Ubuntu 18.04 Installation guide](./install_ubuntu_18.04.md)
```
Comment (hello-atharva) [inline docs/robot_install.md:52]:
```suggestion
```

**Review by hello-atharva — COMMENTED** [review]
Overall looks good, just need to remove catkin-specific scripts. Might need to remove redundant checks inside if-statements (18.04) to make the installation Humble-specific.

**Comment by hello-binit** [comment]
TODO:
 - [ ] 22.04 compressed images for install guide
 - [ ] Workspace y/n prompt disabled for robot install

File: factory/22.04/stretch_create_ament_workspace.sh [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+echo "Setup calibrated robot URDF..."
+ros2 run stretch_calibration update_with_most_recent_calibration >> $REDIRECT_LOGFILE
+
+colcon build >> $REDIRECT_LOGFILE
+
+#TODO:
+#echo "Compiling FUNMAP's Cython code..."
+#cd $CATKIN_WSDIR/src/stretch_ros/stretch_funmap/src/stretch_funmap
+#./compile_cython_code.sh &>> $REDIRECT_LOGFILE
```
Comment (hello-binit) [inline factory/22.04/stretch_create_ament_workspace.sh:86]:
```suggestion
```

**Pull request merged**